provd/main.py

```diff
@@ -33,6 +33,7 @@
 logger = logging.getLogger(__name__)
 
 LOG_FILE_NAME = '/var/log/xivo-provd.log'
+API_VERSION = '0.2'
```

Comment:
Could you explain why 0.2?

Reply:
I chose this version number when I created the api.yml. I think my reason to chose 0.2 was because I added authentication to the API. We could always set it back to 0.1 or 1.0 since there was no version in the URL previously.